pillar/nginx/micromasters_es.sls

```diff
@@ -19,6 +19,11 @@ nginx:
                 - listen:
                     - '[::]:443'
                     - ssl
+                - location /:
```

Comment:
I know it's within the VPC, but do we want to restrict it a bit further for example `/_cluster`?